### DISCUSSION TITLE
What can cause rapid scale shrink?

### DISCUSSION THREAD

**Discussion by mitul93** [stella-cv/stella_vslam#497] [Q&A] [answered]
Hi Stella Community

I am trying to understand the effect of scale drift and its effect on tracking. 
Following image shows trajectory from RICOH THETA Z. After a while, the scale starts to shrink very fast and the system still tracks the features. The trajectory however seems to stuck on the same location and perform only rotation.

![screenshot](https://user-images.githubusercontent.com/4248605/233398432-bc5a0bfc-9ef5-4cac-8e41-3a613defe6fa.png)

I will be thankful if someone will help to understand this problem.

**Comment by ymd-stella** [comment, marked as answer]
Too many keyframes.
